samples/MegaDrone/src/main/cpp/Synth.h

```diff
@@ -60,13 +59,14 @@ class Synth : public IRenderableAudio {
         mOutputStage->renderAudio(audioData, numFrames);
     };
 
-    virtual ~Synth() {}
+    virtual ~Synth() {
+    }
 private:
-
     // Rendering objects
     std::array<Oscillator, kNumOscillators> mOscs;
     Mixer mMixer;
-    std::shared_ptr<IRenderableAudio> mOutputStage;
+    MonoToStereo mConverter = MonoToStereo(&mMixer);
+    IRenderableAudio *mOutputStage; // This will point to either the mixer or converter
```

Comment:
Can this be declared const? The pointer will not change after the constructor.

IRenderableAudio * const mOutputStage; // const pointer to mutable object

Reply:
> Using a const ptr is a little bit messy because of the need to initialize prior to the constructor.
> If it is clearer, we can use a non-const pointer and have it instantiated as before.

I see what you mean. This is messy. Feel free to go back to the original.

Reply:
Added in latest commit